src/network/tc/qdisc.c

```diff
@@ -218,15 +218,15 @@ int qdisc_section_verify(QDisc *qdisc, bool *has_root, bool *has_clsact) {
         if (qdisc->parent == TC_H_ROOT) {
                 if (*has_root)
                         return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                 "%s: More than one root TrafficControlQueueingDiscipline sections are defined. "
-                                                 "Ignoring [TrafficControlQueueingDiscipline] section from line %u.",
+                                                 "%s: More than one root qdisc sections are defined. "
```

Comment:
"More than one [QDisc] section is defined. "
"Ignoring [QDisc] section from line %u."

Reply:
This is not only for [QDisc] section, but also e.g. [NetworkEmulator] section.